app-leechcraft/lc-bittorrent/lc-bittorrent-9999.ebuild

```diff
@@ -14,17 +14,17 @@ IUSE="debug geoip"
 DEPEND="
 	~app-leechcraft/lc-core-${PV}
 	net-libs/libtorrent-rasterbar
-	dev-qt/qtxml:5
 	dev-qt/qtwidgets:5
+	dev-qt/qtxml:5
```

Comment:
Did you check linking? Sure you don't need at least qtcore and qtgui here?

Reply:
Those should be pulled by `qtwidgets`. Should they still be explicitly mentioned in the deps?

Reply:
yes, we never rely on indirect deps, what if on next release qtwidgets drops one of them?